### PR TITLE
Fabo/3833 router loop

### LINKED ISSUE #3833: Router gets into a loop
Sentry Issue: [FRONTEND-CV](https://monitoring.lunie.io:9000/lunie/frontend/issues/562/?referrer=github_integration)

```
RangeError: Maximum call stack size exceeded.
  at f (/bundles/572913a6d9483bf6331e5120bff80f3c175b4da7.js:50:53941)
  at d (/bundles/572913a6d9483bf6331e5120bff80f3c175b4da7.js:50:54066)
  at None (/bundles/572913a6d9483bf6331e5120bff80f3c175b4da7.js:72:124350)
  at None (/bundles/572913a6d9483bf6331e5120bff80f3c175b4da7.js:72:120329)
  at None (/bundles/572913a6d9483bf6331e5120bff80f3c175b4da7.js:74:41601)
...
(45 additional frame(s) were not displayed)
```

### PR DESCRIPTION
**Pull request by faboweb**
Closes #3833

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ on:
 jobs:
   createReleasePR:
     name: createReleasePR
+    runs-on: ubuntu-latest
```
Comment (jbibla) [inline .github/workflows/release.yml:10]: this doesn't seem like it's in the right PR.

**Comment by jbibla** [comment]
looks good to me. 

did you notice anything different after these changes?

what does removing apollo here accomplish?

**Comment by faboweb** [comment]
> what does removing apollo here accomplish?

it wasn't used anymore so I removed the imports

File: .github/workflows/release.yml (hunk 1 above)
Comment (faboweb) [reply .github/workflows/release.yml:10]: true, but it's on develop I think (should be there)

**Pull request merged**